tests/variational/test_inference.py

```diff
@@ -162,22 +159,7 @@ def fit_kwargs(inference, use_minibatch):
 
 
 def test_fit_oo(inference, fit_kwargs, simple_model_data):
-    # Minibatch data can't be extracted into the `observed_data` group in the final InferenceData
```

Comment:
no more issues there?

Reply:
Nope I allow to extract the data in the idata, it extracts the whole data